language/move-prover/spec-lang/src/env.rs

```diff
@@ -416,6 +416,8 @@ pub struct GlobalEnv {
     global_invariants_for_memory: BTreeMap<QualifiedId<StructId>, BTreeSet<GlobalId>>,
     /// A set containing spec functions which are called/used in specs.
     used_spec_funs: BTreeSet<QualifiedId<SpecFunId>>,
+    /// A call graph mapping callers to callees.
+    call_graph: BTreeMap<QualifiedId<SpecFunId>, BTreeSet<QualifiedId<SpecFunId>>>,
```

Comment:
Does this need to be in the GlobalEnv, can it be in the translator instead?

Reply:
Yes translator is a better place to put it.